lib/transaction.py

```diff
@@ -755,8 +756,24 @@ def sign(self, keypairs):
                     secexp = pkey.secret
                     private_key = MySigningKey.from_secret_exponent(secexp, curve = SECP256k1)
                     public_key = private_key.get_verifying_key()
-                    sig = private_key.sign_digest_deterministic(pre_hash, hashfunc=hashlib.sha256, sigencode = ecdsa.util.sigencode_der)
-                    assert public_key.verify_digest(sig, pre_hash, sigdecode = ecdsa.util.sigdecode_der)
+   
+                    use_Schnorr_Sigs = False
+                    Schnorr_Fail = False
+
+                    # Turn on for DEBUG:
+                    # use_Schnorr_Sigs = True
+
+                    if use_Schnorr_Sigs:
```

Comment:
How does this ever get executed in a normal flow? Is the intent to enable this and then try to validate everything with Schnorr and then fall back to ECDSA?

Reply:
Yeah this debug code needs to go. :)

Presumably the transaction class will have a configurable bool that gets set by caller, is what i imagine.